circle_detection/CHANGELOG.rst

```diff
@@ -0,0 +1,157 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package scitos_docking
```

Comment:
This needs to be cleaned up, it was just coped from my repo (which I also need to clean up) :stuck_out_tongue: